js/cell.js

```diff
@@ -45,6 +45,10 @@ Cell.prototype.create = function() {
   }
   this.x = 0;
   this.shift = 0;
+  // default each cell to aria hidden
+  // determination of selected cell
+  // will change this state
+  this.element.setAttribute('aria-hidden', true);
```

Comment:
Good point, I guess I got a bit overwhelmed and wasn't sure I could use that there. That definitely would be the way to go though.